src/Support/Testing/MocksClassLoader.php

```diff
@@ -0,0 +1,65 @@
+<?php namespace Winter\Storm\Support\Testing;
+
+use Winter\Storm\Filesystem\Filesystem;
+use Winter\Storm\Support\ClassLoader;
+
+/**
+ * Helper trait to test classes which use the ClassLoader (ie. extensions).
+ *
+ * This trait will mock the ClassLoader so that it does not require the entire app to be instantiated.
+ */
+trait MocksClassLoader
+{
+    /** @var ClassLoader Mocked ClassLoader instance */
+    protected $classLoader;
+
+    /**
+     * Registers a mock class loader in the autoload stack.
+     *
+     * @param string $basePath
+     * @param string $manifestPath
+     * @return void
+     */
+    protected function registerMockClassLoader($basePath = null, $manifestPath = null)
+    {
+        if (is_null($basePath)) {
+            $basePath = dirname(dirname(__DIR__)) . '/fixtures/classes';
```

Comment:
Doesn't `dirname(__DIR__, 2)` looks more readable?

Reply:
I don't think I've seen the second argument be passed to dirname before in the codebase